File: SMS API/SAMPLE_CODE/CSharp.NET/API21_Sample/WinForm_Sample(HTTP)/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WinForm_Sample_HTTP_ {
	public partial class Form1 : Form {
		private SMSHttp smsHttp = new SMSHttp();

		public Form1() {
			InitializeComponent();
		}

		private void btnLogin_Click(object sender, EventArgs e) {
			if (this.smsHttp.getCredit(this.txtUserID.Text, this.txtPassword.Text)) {
				//取得點數成功
				this.lblCredit.Text = this.smsHttp.Credit.ToString();
				MessageBox.Show("登入成功");
			} else {
				MessageBox.Show("登入失敗");
			}
		}

		private void btnSend_Click(object sender, EventArgs e) {
			if (this.smsHttp.sendSMS(this.txtUserID.Text, this.txtPassword.Text, this.txtSubject.Text, this.txtContent.Text, this.txtPhones.Text, "")) {
				this.lblCredit.Text = this.smsHttp.Credit.ToString();
				MessageBox.Show("傳送成功");
			} else {
				MessageBox.Show("傳送失敗");
			}
		}
	}
}

File: SMS API/SAMPLE_CODE/VB.NET/API21_Sample/WinForm_Sample(HTTP)/Form1.vb
Public Class Form1
    Private smsHttp As SMSHttp = New SMSHttp()

    Private Sub btnLogin_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnLogin.Click
        If smsHttp.getCredit(txtUserID.Text, txtPassword.Text) Then
            ''取得點數成功
            lblCredit.Text = smsHttp.getCreditValue().ToString()
            MessageBox.Show("登入成功")
        Else
            MessageBox.Show("登入失敗")
        End If
    End Sub

    Private Sub btnSend_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnSend.Click
        If smsHttp.sendSMS(txtUserID.Text, txtPassword.Text, txtSubject.Text, txtContent.Text, txtPhones.Text, "") Then
            lblCredit.Text = smsHttp.getCreditValue().ToString()
            MessageBox.Show("傳送成功")
        Else
            MessageBox.Show("傳送失敗")
        End If
    End Sub
End Class
